packages/xod-client/src/editor/messages.js

```diff
@@ -1,3 +1,6 @@
 export const PATCH_FOR_NODE_IS_MISSING = 'Patch for this node is missing.';
 
-export default {};
+export const libInstalled = (libName, version) => `Library ${libName} version ${version} was succesfully installed`;
+
+export const LIB_SUGGESTER_TYPE_TO_BEGIN = 'Type library name to find and install the library. E.G. `xod/core`';
```

Comment:
A dangerous example for a user. Maybe just: “Type owner/libname to find a library”?